packages/shadow-root-web-component.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
```

Comment:
This is only used as a test-environment in core-suggest, created a pre-transpiled file to ensure support in the web-driver.